idf_build_apps/main.py

```diff
@@ -594,6 +605,7 @@ def main():
         manifest_rootpath=args.manifest_rootpath,
         ignore_component_dependencies_file_patterns=args.ignore_component_dependencies_file_patterns,
         depends_on_files=args.depends_on_files,
+        sdkconfig_defaults=args.sdkconfig_defaults,
```

Comment:
I get following error if I try to find an application without `sdkconfig.defaults`
```
Traceback (most recent call last):
  File "runpy.py", line 194, in _run_module_as_main
  File "runpy.py", line 87, in _run_code
  File "C:\github_repos\idf-build-apps\idf_build_apps\__main__.py", line 9, in <module>
    main()
  File "C:\github_repos\idf-build-apps\idf_build_apps\main.py", line 608, in main
    sdkconfig_defaults=args.sdkconfig_defaults,
AttributeError: 'Namespace' object has no attribute 'sdkconfig_defaults'
```